recipes/alfa/meta.yaml

```diff
@@ -39,4 +42,6 @@ about:
 
 extra:
   recipe-maintainers:
-    - fgypas
+    - fgypas 
+  additional-platforms:
+    - linux-aarch64
```

Comment:
The package is `noarch`. No need of `linux-aarch64`

Reply:
Let's see what people think about https://github.com/bioconda/bioconda-recipes/pull/46425#issuecomment-1999101539